### PR TITLE
Fix: issue#708, load data cannot work with relative file path

### PR DESCRIPTION
**Pull request by broccoliSpicy**
Add absolute file path check to the function handleLoadData.

**What type of PR is this?**

- [ ] API-change
- BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

issue #708 

**What this PR does / why we need it:**

Not Available

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

**Comment by broccoliSpicy** [comment]
I can't check the build status on CentOS/x64 as I don't have a CentOS machine, but my build on Ubuntu was fine.

File: pkg/frontend/mysql_cmd_executor.go [hunk 1]
```diff
@@ -528,9 +529,12 @@ func (mce *MysqlCmdExecutor) handleLoadData(load *tree.Load) error {
 	/*
 		check file
 	*/
+	if !filepath.IsAbs(load.File) {
+		load.File, _ = filepath.Abs(load.File)
```
Comment (daviszhen) [inline pkg/frontend/mysql_cmd_executor.go:533]: It is inappropriate to ignore the **err** here. The err should be pitched.

**Review by daviszhen — COMMENTED** [review]
I encourage you to add some unit test for the relative file path in mysql_cmd_executor_test.go. @broccoliSpicy

**Comment by daviszhen** [comment]
> I can't check the build status on CentOS/x64 as I don't have a CentOS machine, but my build on Ubuntu was fine.

It is fine in linux

**Comment by broccoliSpicy** [comment]
I think the issue#708 is perhaps a false issue and described the details under the original post of [Issue#708](https://github.com/matrixorigin/matrixone/issues/708)

**Pull request closed without merging**